src/ai/open_ai.rs

```diff
@@ -40,70 +40,142 @@ pub async fn generate_message(
         HeaderValue::from_str("application/json").unwrap(),
     );
 
+    let json_body = serde_json::json!({
+        "model": model,
+        "temperature": 0.2,
+        "messages": messages,
+    });
+
+    let response = client
+        .post(url)
+        .headers(headers)
+        .body(serde_json::to_string(&json_body).unwrap())
+        .send()
+        .await;
+
+    match response {
+        Ok(r) => r.json::<Value>().await,
+        Err(e) => {
+            println!("ERROR: Generating the message {}", e.to_string());
+            Ok(Value::Null)
+        }
+    }
+}
+
+pub fn get_prompt_for_commit(
```

Comment:
I've created a function to get the prompt since we can use a different prompt to create pull requests